google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiManager.java

```diff
@@ -57,6 +60,9 @@ private CloudApiManager() {}
   /**
    * Enables the supplied set of {@link CloudLibrary CloudLibraries} on GCP.
    *
+   * <p>Configures the {@link ProgressIndicator} to display the progress of the tasks. Also notifies
+   * the user of the success / failure of API enablement via messages on the event log.
+   *
    * @param libraries the set of {@link CloudLibrary CloudLibraries} to enable on GCP
    * @param cloudProject the {@link CloudProject CloudProject} on which to enable the APIs
```

Comment:
nit: since we provide parameters doc, shall we add `project` parameter short description as well?

Reply:
oops. fixed.